packages/core/CalendarManager.ts

```diff
@@ -280,7 +280,7 @@ export const createEvent = async (
         if (error?.calError) {
           calError = error.calError;
         }
-        log.error("createEvent failed", error, calEvent);
+        log.error("createEvent failed", JSON.stringify(error), calEvent);
```

Comment:
Makes it print the error in the same line and thus easy to debug in axiom as well as locally.